pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -126,6 +126,7 @@ def __init__(
         self._model_averaging_period = model_averaging_period
         self._pids: Optional[List[int]] = None
         self._sync_dir: Optional[str] = None
+        self._has_called_call_children_scripts: bool = False
```

Comment:
if we are going with this, let's name it appropriately to not be misleading :)

```suggestion
        self._rank_0_has_called_call_children_scripts: bool = False
```